pyffs/util.py

```diff
@@ -425,3 +426,34 @@ def _create_modulation_vectors(N_s, N_FS, T, T_c, mod=None):
         E_2 = mod.r_[mod.arange(start=0, stop=M), mod.arange(start=-M, stop=0)]
 
     return B_1 ** E_1, B_2 ** (N * E_2)
+
+
+def ffsn_shift(x, idx):
```

Comment:
Don't understand what this does?

Reply:
It's like `fftshift`: https://numpy.org/doc/stable/reference/generated/numpy.fft.fftshift.html
For higher dimensions, it gets clunky to re-order the samples with the output of `ffsn_sample`. So I thought a utility function could allow for more compact code. Could also provide the same args as those provided to `ffsn_sample` so that the indices are computed internally, but then that's more arguments...